Sources/Publish/Internal/PublishingPipeline.swift

```diff
@@ -81,9 +82,10 @@ private extension PublishingPipeline {
     }
 
     func setUpFolders(withExplicitRootPath path: Path?,
+                      explicitOutputPath outputPath: Path?,
                       shouldEmptyOutputFolder: Bool) throws -> Folder.Group {
         let root = try resolveRootFolder(withExplicitPath: path)
-        let outputFolderName = "Output"
+        let outputFolderName = outputPath?.string ?? "Output"
```

Comment:
Since `outputPath` is a `Path`, this can no longer be treated as a folder name, and must also be treated as a `Path`. I suggest renaming this to `outputFolderPath`, and using `outputPath ?? "Output"` as its expression.

Reply:
Yes, that's what we did in the end. Let's close this PR. Thanks a lot.